collections/sea-ice-index.yaml

```diff
@@ -11,6 +11,7 @@ AdditionalInfoExternal:
   Title: Additional info
   Path: sea-ice-index/README.MD
 Image: sea-ice-index/sea-ice.png
+EDCBrowser: https://browser.eurodatacube.com/?zoom=5&lat=71.20137&lng=-28.9579&collectionId=sea-ice-index&customVisualizationSelected=true&evalscript=Ly9WRVJTSU9OPTMKZnVuY3Rpb24gc2V0dXAoKSB7CiAgcmV0dXJuIHsKICAgIGlucHV0OiBbImNvbmNlbnRyYXRpb24iLCJleHRlbnQiLCJkYXRhTWFzayJdLAogICAgb3V0cHV0OiB7IGJhbmRzOiAzLCBzYW1wbGVUeXBlOiAiQVVUTyIgfQogIH07Cn0KCmZ1bmN0aW9uIGV2YWx1YXRlUGl4ZWwoc2FtcGxlKSB7CiAgcmV0dXJuIFsyLjUqc2FtcGxlLmNvbmNlbnRyYXRpb24sMi41KnNhbXBsZS5leHRlbnQsMi41KnNhbXBsZS5kYXRhTWFza107fQ%3D%3D&type=sentinel-hub-edc&fromTime=2020-01-31T00%3A00%3A00.000Z&toTime=2021-01-31T00%3A00%3A00.000Z#custom-composite
```

Comment:
This doesn't display anything.

Reply:
Try to fix it by providing evalscripts and add configuration to ymal.

Reply:
This does work however because of the complexity of the script and because it's creodias it takes 4-5 minutes for the default timespan (1 year) to load the data. I think that's not really ideal and the user will certainly give up on that. I think we should either try to come up with a more simple script or modify the default timespan for this collection (has to be done in the EDC Browser code). If you select only 1 day or a few days timespan it works fine (e.g. [here](https://webdev.sentinel-hub.com/edc-browser/?zoom=4&lat=-74.07992&lng=178.50586&collectionId=sea-ice-index&customVisualizationSelected=true&evalscript=Ly9WRVJTSU9OPTMKZnVuY3Rpb24gc2V0dXAoKSB7CiAgICByZXR1cm4gewogICAgICBpbnB1dDogWyJjb25jZW50cmF0aW9uIiwgImRhdGFNYXNrIl0sCiAgICAgIG91dHB1dDogeyBiYW5kczogNCwgc2FtcGxlVHlwZTogIkFVVE8iIH0sCiAgICAgIG1vc2FpY2tpbmc6ICJUSUxFIgogICAgfTsKICB9CgogIC8vIFNldCBjb2xvcmJhcgogIHZhciBjYmFyID0gW1swLCIweEVCRjlGRiJdLAogICAgICAgICAgICAgWzEwLCAiMHhEMUU2RjciXSwKICAgICAgICAgICAgIFsyMCwgIjB4QjdENEVGIl0sCiAgICAgICAgICAgICBbMzAsICIweDlEQzFFNyJdLAogICAgICAgICAgICAgWzQwLCIweDgzQUZERiJdLAogICAgICAgICAgICAgWzUwLCAiMHg2ODlDRDYiXSwKICAgICAgICAgICAgIFs2MCwgIjB4NEU4QUNFIl0sCiAgICAgICAgICAgICBbNzAsICIweDM0NzdDNiJdLAogICAgICAgICAgICAgWzgwLCAiMHgxQTY1QkUiXSwKICAgICAgICAgICAgIFs5MCwgIjB4MDA1MkI2Il1dOwoKICAvLyBTZXQgcmFtcAogIGNvbnN0IHZpc3VhbGl6ZXIgPSBuZXcgQ29sb3JSYW1wVmlzdWFsaXplcihjYmFyKTsKCiAgZnVuY3Rpb24gZXZhbHVhdGVQaXhlbChzYW1wbGVzKSB7CgogICAgaWYgKHNhbXBsZXNbMF0uY29uY2VudHJhdGlvbiA9PSAwICYmIHNhbXBsZXNbMV0uY29uY2VudHJhdGlvbiAhPT0gMCl7CiAgICAgICAgdmFyIGRhdGEgPSAxOwogICAgICAgIHZhciBzYW1wbGUgPSBzYW1wbGVzWzFdOwogICAgfSBlbHNlIGlmICgoc2FtcGxlc1sxXS5jb25jZW50cmF0aW9uID09IDAgJiYgc2FtcGxlc1swXS5jb25jZW50cmF0aW9uICE9PSAwKSl7CiAgICAgICAgdmFyIGRhdGEgPSAxOwogICAgICAgIHZhciBzYW1wbGUgPSBzYW1wbGVzWzBdOwogICAgfSBlbHNlIHsKICAgICAgICB2YXIgZGF0YSA9IDA7CiAgICB9CgogICAgaWYgKGRhdGEgIT09IDApIHsKICAgICAgICB2YXIgcmdiID0gdmlzdWFsaXplci5wcm9jZXNzKHNhbXBsZS5jb25jZW50cmF0aW9uIC8gMTApOwogICAgICAgIHJnYi5wdXNoKDEpOwoKICAgICAgICBpZiAoc2FtcGxlLmNvbmNlbnRyYXRpb24gPT0gMjU0MCl7CiAgICAgICAgICByZXR1cm4gWzEsIDEsIDEsIDFdOwogICAgICAgIH0gZWxzZSBpZiAoc2FtcGxlLmNvbmNlbnRyYXRpb24gPT0gMjUzMCl7CiAgICAgICAgICByZXR1cm4gWzAuNSwgMC41LCAwLjUsIDFdOwogICAgICAgIH0gZWxzZSB7CiAgICAgICAgICByZXR1cm4gcmdiOwogICAgICAgIH0KICAgIH0gZWxzZSB7CiAgICAgICAgcmV0dXJuIFswLCAwLCAwLCAwXTsKCiAgICB9Cn0g&type=sentinel-hub-edc&fromTime=2021-05-03T00%3A00%3A00.000Z&toTime=2021-05-03T23%3A59%3A59.999Z#custom-script))